### PR TITLE
Revision

### PR DESCRIPTION
**Pull request by markedwardmurray**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,340 +1,133 @@
...
 
-######Example (in C)
+[apple_oop_guide_intro]: https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/OOP_ObjC/Introduction/Introduction.html#//apple_ref/doc/uid/TP40005149-CH1-SW2
 
-```
-void DisplayWhoIAm(){
-    printf("I am the writer of this tutorial!");
-}
-```
+Since all computer code boils down to binary it's natural to wonder, "how can a string of ones and zeroes be referred to as an 'object'?" The use of the word "object" is an abstraction of thought. An "object" in code has no more physical form than does a word in any human language. Sure, words have physical representations: speaking a word causes air to vibrate in a sound wave, ink on a page can be shaped into symbols that represent the word, a meaning can be pointed at or mimed out; but none of these are the word itself. Human language is a system of abstraction: it communicates the *idea* of a thing, but not the thing itself.
```
Comment (jmburges) [inline README.md]: remember, they don't know what an "object" in the programming sense is yet! Warm them up a bit to that.

Comment (jmburges) [inline README.md]: that being said WHOA. I love this paragraph. Really beautiful

**Comment by markedwardmurray** [comment]
@jmburges need a second look at this one, too.

**Comment by jmburges** [comment]
Upload those images to S3 and :shipit:

**Comment by markedwardmurray** [comment]
is there a reason that we can't trust Wikipedia to host them? just because we can't guarantee they won't be taken down at some point?

**Pull request merged**